### ISSUE TITLE
Navigating in the editor is slow with a screen reader

### ISSUE THREAD

**Issue by cary-rowen** [microsoft/vscode#163505]
Type: <b>Performance Issue</b>

I'm using an NVDA screen reader and typing code in the VS Code editor has speech lag.
When I press keys on the keyboard I often cannot hear the typed characters very quickly.
But if my input is slowed down, I can hear it.

VS Code version: Code 1.72.1 (129500ee4c8ab7263461ffe327268ba56b9f210d, 2022-10-10T17:22:48.346Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.88GB (9.46GB free)|
|Process Argv|--disable-features=CalculateNativeWinOcclusion --crash-reporter-id 08ab617a-b493-4182-980e-7383997e95a3 --crash-reporter-id 08ab617a-b493-4182-980e-7383997e95a3|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
    3	   162	 14256	code main
    0	    75	   444	   utility-network-service
    0	   169	  2676	   gpu-process
    0	    72	  7992	   crashpad-handler
    0	   212	  9872	   window (dataStructure (工作区) - Visual Studio Code)
    0	    78	 11404	   utility
    0	    24	 12132	   "C:\Program Files (x86)\SogouInput\12.5.0.6558\SGTool.exe" -daemon
    0	    95	 13476	   issue-reporter
    0	   155	 15668	   extensionHost
    0	    92	 14204	     "C:\Users\cary\AppData\Local\Programs\Microsoft VS Code\Code.exe" --ms-enable-electron-run-as-node "c:\Users\cary\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\json-language-features\server\dist\node\jsonServerMain" --node-ipc --clientProcessId=15668
    0	    22	 18456	     c:\Users\cary\.vscode\extensions\ms-vscode.cpptools-1.12.4-win32-x64\bin\cpptools.exe
    0	    11	 17112	       console-window-host (Windows internal process)
    0	     4	 19240	       "c:\Users\cary\.vscode\extensions\ms-vscode.cpptools-1.12.4-win32-x64\bin\cpptools.exe"
    0	    25	  8612	         c:\Users\cary\.vscode\extensions\ms-vscode.cpptools-1.12.4-win32-x64/bin/cpptools-srv.exe 18456 {5750E01A-955C-4C3C-AEDF-A031C4F78F2D}
    0	    11	  2652	           console-window-host (Windows internal process)
    0	   125	 19376	   shared-process
    0	    87	  2776	     "C:\Users\cary\AppData\Local\Programs\Microsoft VS Code\Code.exe" --ms-enable-electron-run-as-node "c:\Users\cary\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork" ms-vscode.cppdbg "{\"common.vscodemachineid\":\"694208feb7ed776717124b8b373b10d67bfa55b4fd4080ace0059213748a7fb9\",\"common.vscodesessionid\":\"1ba88fbf-a7a2-44e7-afd7-3cc9d7fd08f91665624789916\"}" AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217
    0	    87	  4144	     ptyHost
    0	    84	 17520	     fileWatcher
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (dataStructure (工作区) - Visual Studio Code)
|    Folder (dataStructure): 8 files
|      File types: json(2) exe(1) c(1) code-workspace(1)
|      Conf files: launch.json(1) tasks.json(1)
|      Launch Configs: cppdbg;
```

</details>
<details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
doxdocgen|csc|1.4.0
vscode-language-pack-zh-hans|MS-|1.72.10121008
remote-containers|ms-|0.255.3
remote-ssh|ms-|0.90.1
remote-wsl|ms-|0.72.0
cmake-tools|ms-|1.12.27
cpptools|ms-|1.12.4
cpptools-extension-pack|ms-|1.3.0
cmake|twx|0.0.17

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
2144e591:30553903
pyindex848cf:30577861
nodejswelcome1:30587005
40g7c324:30573242

```

</details>

**Comment by meganrogge** [comment]
Does the performance improve if you run the command `Extensions: Disable All Installed Extensions`?

**Comment by cary-rowen** [comment]
This situation is accidental, I'll keep an eye on it, and I'll close this issue without further information.

**Comment by cary-rowen** [comment]
I haven't found a step that can be reproduced stably, which may also be related to NVDA.
